src/Controller/StoreApiController.php

```diff
@@ -32,9 +32,12 @@
 use Adyen\Shopware\Service\ConfigurationService;
```

Comment:
Maybe it's time to consider splitting this up into different controllers

Reply:
Thanks @peterojo,  created a separate ticket about it